docs/Aks-Cluster-Automation.md

```diff
@@ -0,0 +1,58 @@
+# AKS Cluster Restart Automation
+
+The instructions below describe how to implement an Azure Automation Runbook that will automate and schedule stopping and starting the AKS clusters in the ngsa-asb setup. Though the runbook enables stopping and restarting the AKS clusters, the scheduled jobs created in this section only restart the application gateways.
```

Comment:
Please update last part of this paragraph to reflect what is going to be restarted is the cluster instead of application gateway.  
Currently it reads as "the scheduled jobs created in this section only restart the application gateways."